database/wrapper/player_wrapper.py

```diff
@@ -27,3 +28,31 @@ def get_random_player(session):
         logger.warning(e)
         player = create_default_player()
     return player
+
+
+class PlayerWrapper:
+    def __init__(self, limit=20):
+        self.limit = limit
+
+    def get_player_games(self, session, id, id_list=False):
+        if id_list:

```

Comment:
Can we not do `isinstance(id, list)` instead? Then we wouldn't have issues later if we try to compare a list to an ID. I understand that's less verbose (...but we could just write a docstring).